### PR TITLE
Add arbitrum Gyro2V2PoolFactory

### PR DESCRIPTION
**Pull request by joshguha**

### CODE REVIEW

File: networks.yaml [hunk 1]
```diff
@@ -483,6 +483,9 @@ arbitrum:
   FXPoolDeployer:
     address: "0x0bd5EC16658346eeCd5dE8c704a38Efe02B5DA69"
     startBlock: 183603903
+  Gyro2V2PoolFactory:
+    address: "0x8342b910815b0127c98e7717d4276c1d393478b6"
```
Comment (mendesfabio) [inline networks.yaml:487]: I don't see any events emitted by this contract -- this is probably the implementation, can you verify it?

**Review by mendesfabio — CHANGES_REQUESTED** [review]

File: networks.yaml (hunk 1 above)
Comment (mendesfabio) [reply networks.yaml:487]: I believe this is the factory https://arbiscan.io/address/0x7a36527a02d96693b0af2b70421f952816a4a088

Comment (joshguha) [reply networks.yaml]: Yes @mendesfabio you're completely right! Updated that now

File: networks.yaml [hunk 2]
```diff
@@ -483,6 +483,9 @@ arbitrum:
   FXPoolDeployer:
     address: "0x0bd5EC16658346eeCd5dE8c704a38Efe02B5DA69"
     startBlock: 183603903
+  Gyro2V2PoolFactory:
+    address: "0x7a36527A02d96693b0AF2b70421F952816a4a088"
+    startBlock: 194366254
```
Comment (mendesfabio) [inline networks.yaml:488]:
```suggestion
    startBlock: 194367293
```

**Pull request merged**